### PR TITLE
Added mock dto requests for v5 dtos

### PR DESCRIPTION
**Pull request by edwardlach**
Added mock dto v5 classes to be used in testing the Admin App

### CODE REVIEW

File: src/lib/dto/v5/abstract-sensor-session.dto.v5.ts [hunk 1]
```diff
@@ -1,11 +1,11 @@
-import { AbstractUserOwnedResponseV5 } from './abstract-user-owned.dto.v5';
 import { AbstractSensorEventCollectionResponseSummaryV5 } from './abstract-sensor-event.dto.v5';
+import { AbstractUserOwnedResponseV5 } from './abstract-user-owned.dto.v5';
 
 export interface AbstractSensorSessionResponseV5 extends AbstractUserOwnedResponseV5 {
 	competitionLevelUuid: string;
 	userUuid: string;
 	extensionType: string;
-	extensionDetails: { [name: string]: object };
+	extensionDetails: { [name: string]: any };
```
Comment (edrobotij) [inline src/lib/dto/v5/abstract-sensor-session.dto.v5.ts:8]: I would avoid using the type `any` in favor of `unknown`. The type `any` won't produce any type errors which makes it tougher to debug. There's some further explanation here https://dmitripavlutin.com/typescript-unknown-vs-any/

If you found this in code I wrote, then boo me!

File: src/lib/dto/v5/swing.dto.v5.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { AbstractSensorEventResponseV5 } from './abstract-sensor-event.dto.v5';
-import { SwingMetricDtoV5 } from './swing-metric.dto.v5';
 import { CollectionResponseV5 } from './collection.dto.v5';
+import { SwingMetricDtoV5 } from './swing-metric.dto.v5';
```
Comment (edrobotij) [inline src/lib/dto/v5/swing.dto.v5.ts:3]: Should probably keep the naming consistent with `PitchingMetricDTOV5` (capital DTO).

**Review by edrobotij — CHANGES_REQUESTED** [review]
Just a couple of minor things, other than that it looks good! No need to request another review after the changes.

File: src/lib/dto/v5/swing.dto.v5.ts (hunk 2 above)
Comment (edwardlach) [reply src/lib/dto/v5/swing.dto.v5.ts:3]: For sure!

File: src/lib/dto/v5/abstract-sensor-session.dto.v5.ts (hunk 1 above)
Comment (edwardlach) [reply src/lib/dto/v5/abstract-sensor-session.dto.v5.ts:8]: Good to know, I didn't know unknown was a type!  Looks like you have extension details as a string type in the v3 code.  I was trying to convert from the Java V5 DTOs.  Should extensionDetails be a string or an object on the client?

**Pull request merged**